src/components/Profile/Item.tsx

```diff
@@ -2,16 +2,40 @@ import React from 'react';
 
 export default function Item(props) {
   return (
-    <div className="profile-item" onClick={() => props.goToChat(0)}>
+    <div className="profile-item" onClick={() => props.goToChat(props.index)}>
       {props.checkmark ? (
-        <div className="profile-item-checkmark" />
+        <div className="profile-item-checkmark">
+          <img
+            src="icons/check-mark-3-512.png"
+            alt="checkmark"
+            style={{
```

Comment:
Inline styling for multiple rows should probably have its own class in the css. Otherwise it creates inconsistency within the project.

Reply:
fixed.